CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
+### Changes for Sprint 66
```

Comment:
@JPrause we jump from three pounds to two pounds to three pounds, on purpose? justtttt wonderrrringggg 🤔